docs/preset-env.md

```diff
@@ -229,40 +229,45 @@ This option is useful for "blacklisting" a transform like `@babel/plugin-transfo
 
 `"usage"` | `"entry"` | `false`, defaults to `false`.
 
-This option configures how `@babel/preset-env` handles polyfills.
+This option configures how `@babel/preset-env` imports various polyfills in your generated code.
 
-When either the `usage` or `entry` options are used, `@babel-preset-env` will add direct references to `core-js` modules as bare imports (or requires). This means `core-js` will be resolved relative to the file itself and needs to be accessible.
+Specifically, this option supports the following external polyfill packages:
 
-Since `@babel/polyfill` was deprecated in 7.4.0, we recommend directly adding `core-js` and setting the version via the [`corejs`](#corejs) option.
+- `core-js`: For standard JS built-ins
+- `regenerator`: For generator and `async`/`await` support
```

Comment:
`regenerator` is not a polyfill.

Reply:
I don't think I say it is..? Unless you're saying I'm implying that it is -- in which case, could you suggest a better way of describing it?

Reply:
> I don't think I say it is..?

> ... the following external polyfill packages:
> - `core-js`: ...
> - `regenerator`: ...
---
It's runtime, not a polyfill.

Reply:
Gotcha, thank you.